lldb/include/lldb/Interpreter/CommandReturnObject.h

```diff
@@ -10,7 +10,7 @@
 #define LLDB_INTERPRETER_COMMANDRETURNOBJECT_H
 
 #include "lldb/Host/StreamFile.h"
-#include "lldb/Utility/DiagnosticsRendering.h"
+#include "lldb/Host/common/DiagnosticsRendering.h"
```

Comment:
Unrelated to this PR, but I'm surprised to see that the subdirectories in `Host` are all lower-case 🤷‍♂️

Reply:
Not sure what's the reason (if there is one).

I assume it would be too cumbersome to change as well now?